### PR TITLE
[zh-cn] sync networking/virtual-ips.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
related: https://github.com/kubernetes/website/issues/39987 task-6

### CODE REVIEW

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -77,15 +79,18 @@ to use as-is.
...
+three replicas. Those replicas are
 fungible&mdash;frontends do not care which backend they use.  While the actual Pods that
 compose the backend set may change, the frontend clients should not need to be aware of that,
 nor should they need to keep track of the set of backends themselves.
 -->
 <a id="example"></a>
 本文中的一些细节会引用这样一个例子：
-运行了 3 个 Pod 副本的无状态图像处理后端工作负载。
+运行了 3 个 {{< glossary_tooltip text="Pod" term_id="pod" >}}
+副本的无状态图像处理后端工作负载。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:93]:
```suggestion
副本的无状态镜像处理后端工作负载。
```

Comment (tengqm) [reply content/zh-cn/docs/reference/networking/virtual-ips.md:93]: 这里的 image 确实是“图像”的意思

**Comment by windsonsea** [comment]
/approve

Need to trigger the CI again.

**Review by ydFu — APPROVED** [review]
This PR submission assists in resolving some of the subtasks in "Reference files to sync for 1.26 #39987". Thank you.

/lgtm

**Pull request merged**